internal/helm/integration_test.go

```diff
@@ -81,7 +81,7 @@ func testAddPlatformRepository(home string) func(t *testing.T) {
 		for i := 0; i < 2; i++ {
 			err := helmService.AddRepositoryIfNotExists(helm.Repository{
```

Comment:
I beleive we are missing this call from the code now that a custom chart is being used

Reply:
Yeah, Poke noted the same, I'm going to recheck the official->banzaicloud chart change PR(s) and make sure restore the repository addition.

Reply:
The GitHub PR search didn't pick up the [PR](https://github.com/banzaicloud/pipeline/pull/2905) where the repository addition was removed - possibly because it did not contain the keyword `kubefed`.

I found it by walking through the blame of the `internal/cmd/config.go` file to find the commit where the Banzai chart was introduced in place of the official.

Reply:
I believe I fixed it [here](https://github.com/banzaicloud/pipeline/compare/d22ce6ac6f021148caee0dcdf56a4cd2f7fecaa8..f0d39f3108640a1e1e519e74ef5aab555b5ebdfe), but please check it for the right content/placement.

Reply:
I think it should do it, at least some e2e tests will tell 👍